plugins/hls-eval-plugin/test/Eval.hs

```diff
@@ -147,15 +150,25 @@ tests =
         , testCase
             "Prelude has no special treatment, it is imported as stated in the module"
             $ goldenTest "TPrelude.hs"
+        , testCase
+            "Don't panic on {-# UNPACK #-} pragma"
+            $ goldenTest "TUNPACK.hs"
+        , testCase
+            "Can handle eval inside nested comment properly"
+            $ goldenTest "TNested.hs"
         , testCase "Test on last line insert results correctly" $ do
             runSession hlsCommand fullCaps evalPath $
                 liftIO $ do
                     let mdl = "TLastLine.hs"
                     -- Write the test file, to make sure that it has no final line return
-                    writeFile (evalPath </> mdl) $ "module TLastLine where\n\n-- >>> take 3 [1..]"
+                    writeFile (evalPath </> mdl) "module TLastLine where\n\n-- >>> take 3 [1..]"
                     goldenTest mdl
 #if __GLASGOW_HASKELL__ >= 808
-            , testCase "CPP support" $ goldenTest "TCPP.hs"
+            ,
+#if mingw32_HOST_OS
```

Comment:
Maybe we could add a dedicated function to mark tests broken for windows like https://github.com/haskell/haskell-language-server/blob/master/test/utils/Test/Hls/Util.hs#L119
We can use `extra` function `isWindows`: https://hackage.haskell.org/package/extra-1.7.9/docs/System-Info-Extra.html#v:isWindows

Reply:
Anyway i will try to take a look in my local windows env